components/Projects/FiltersBar/FiltersBar.tsx

```diff
@@ -85,12 +87,13 @@ const FiltersBar: React.FC = () => {
                 <>
                   <div
                     ref={filterRef}
-                    className="z-[101] absolute -bottom-[23.5rem] md:-bottom-[14rem] right-2 px-[34px] py-[27px] rounded-md border border-blue-600 min-w-[400px] md:min-w-0 md:w-[863px] h-[360px] md:h-[209px] p-5 bg-gray-50 dark:bg-gray-600"
+                    className="z-[101] absolute -bottom-[23.5rem] md:-bottom-[14rem] right-2 px-[34px] py-[27px] rounded-md border border-blue-600 min-w-[400px] md:min-w-0 md:w-[863px] min-h-[260px] md:h-[209px] p-5 bg-gray-50 dark:bg-gray-600"
                   >
                     <div className="flex flex-col gap-[22px]">
                       <h3 className="text-base font-bold leading-normal">
```

Comment:
Not sure its related to this PR, but on my screen right now the filter looks like this : 

![image](https://github.com/Maakaf/maakaf-website/assets/34707669/3fa06b91-4a7b-4a91-864e-10b70a8d6de3)

We can replace the container pixles size with something more responsive like this for example: 
` <div className="flex flex-col gap-1">`

And it will contain all the items : 

![image](https://github.com/Maakaf/maakaf-website/assets/34707669/f2117449-3eb3-4ed8-bce3-1ce6dbd83ee4)

Reply:
Hmm...I changed to min-h-[260px] should give proper containment regardless of tags amount

Reply:
please double check this now.

Reply:
I dont see any difference, I checked your branch and `git pull` (I can see your changes) : 

![image](https://github.com/Maakaf/maakaf-website/assets/34707669/e56fdaea-189a-4996-8540-697d02d5522a)

Can you try to give your container non pixles height? maybe it will be more consistent across different devices?